packages/manager/modules/carrier-sip/src/cdr/cdr.controller.js

```diff
@@ -0,0 +1,84 @@
+import capitalize from 'lodash/capitalize';
+import get from 'lodash/get';
+
+import { NUMBER_OF_YEARS, POLLER_TIMEOUT } from './cdr.constants';
+
+export default class CarrierSipCdrCtrl {
+  /* @ngInject */
+  constructor(
+    $timeout,
+    $translate,
+    $window,
+    OvhApiTelephony,
+    TucToast,
+  ) {
+    this.$timeout = $timeout;
+    this.$translate = $translate;
+    this.$window = $window;
+    this.OvhApiTelephony = OvhApiTelephony;
+    this.TucToast = TucToast;
+  }
+
+  $onInit() {
+    this.months = moment.months().map(capitalize);
+    this.month = null;
+    this.years = [];
+    this.year = null;
+    this.poller = null;
+    const currentYear = moment().year();
+    for (let i = 0; i < NUMBER_OF_YEARS; i += 1) {
+      this.years.push(currentYear - i);
```

Comment:
`_.range` ?

Reply:
To facilitate code readability I can use this function yes.

Reply:
It's done.